backend/de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/model/PO.java

```diff
@@ -5473,6 +5480,58 @@ public final int getLoadCount()
 		return m_loadCount;
 	}
 
+	public final void setCustomColumns(final Map<String, Object> valuesByColumnName)
+	{
+		valuesByColumnName.keySet()
```

Comment:
instead of Map.keySet().forEach, you could use java.util.Map#forEach which takes a BiConsumer<? super K, ? super V> which will receive key and value all together.